sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -647,7 +647,7 @@ export class EventHubClient {
    * @throws {Error} Thrown if the underlying connection has been closed, create a new EventHubClient.
    * @throws {AbortError} Thrown if the operation is cancelled via the abortSignal.
    */
-  async getPartitionIds(options: GetPartitionIdsOptions = {}): Promise<Array<string>> {
```

Comment:
I dont get why is the option mandatory

Reply:
I'm trying to remove (from internal classes) any "defaults" where it makes sense. 

The biggest problem is that I don't want to accidentally omit a parameter (like I just did in another area) and not get a compile error.